# Molecular phylogenetics of the tribe Oryzomyini using multiple markers.

J. Delton Hanson and R. Nelson Platt II.

Taxonomic relationships of Oryzomyini rodents were examined to test phylogenetic hypotheses for the tribe, and the placement of currently accepted genera.  To accomplish this, taxonomic sampling and number of genes examined were increased relative to previous examinations.  Phylogenetic relationships were estimated with maximum likelihood and Bayesian methods.  A total of 84 individual samples were examined for the 1st gene (_Rbp3_ – interphotoreceptor retinoid binding protein).  Although this was an increase of individuals from the most recent molecular work published on the whole tribe, the phylogenetic resolution provided did not increase accordingly.  Phylogenetic resolution was also not increased with a 2nd nuclear gene (alcohol dehydrogenase). When the tribe was examined using a mitochondrial gene (cytochrome-_b_), support was observed almost exclusively at recent nodes, however when the mitochondrial and nuclear datasets were combined phylogenetic resolution was observed at most nodes.  The multi-gene, combined analyses supported a monophyletic tribe Oryzomyini consistent with the most recent arrangement of the tribe.  Additionally, all 10 genera elevated from _Oryzomys_ (_Aegialomys_, _Cerradomys_, _Eremoryzomys_, _Euryoryzomys_, _Hylaeamys_, _Mindomys_, _Nephlomys_, _Oreoryzomys_, _Sooretamys_, and _Transandinomys_) were examined, as were a previously unexamined taxon (_Microakodontomys_), and 2 genera recently named (_Drymoreomys_ and _Tanyuromys_).  Genetic divergence within and among some generic groups suggested that further examination is necessary to infer their phylogenetic status.

#### Submitted to TTUOP (02-26-2019)
---
### NOTES:

Supplemental Data 1. Primer seqeunces used in PCR amplifications and sequencing reactions.

Supplemental Data 2.—Phylogenetic tree obtained from the Bayesian analysis of DNA sequences for  interphotoreceptor retinoid binding protein.  Asterisks above branches represent clade probabilities values >95.  Bootstrap support values >50 (obtained from a parsimony analysis of the same dataset) are shown below branches.

Supplemental Data 3.—Phylogenetic tree obtained from the Bayesian analysis of DNA sequences for alcohol dehydrogenase intron 2 (including informative indels).  Asterisks above branches represent clade probabilities values >95.  Bootstrap support values >50 (obtained from a parsimony analysis of the same dataset) are shown below branches.

Supplemental Data 4.—Phylogenetic tree obtained from the Bayesian analysis of DNA sequences for  cytochrome-b datasets.  Asterisks above branches represent clade probabilities values >95.  Bootstrap support values >50 (obtained from a parsimony analysis of the same dataset) are shown below branches.
